src/db.js

```diff
@@ -14,7 +14,7 @@ export type ID = string
 
 export type Profile = {
   id: ID, // base64-encoded public key
-  avatar?: ?string,
+  avatarSwarmHash?: ?string,
```

Comment:
Do we need this change? Can't we kee the field name to `avatar` and add a comment it is a swarm hash?

Reply:
thought it might be better to be more explicit in the naming but can just add a comment if you think that makes it clear enough

Reply:
Yes I'd rather not start using field names as types unless we have multiple variants ex `avatarSwarmHash` and `avatarURL`, for now just `avatar` is enough I think